code/modules/hydroponics/grown/peas.dm

```diff
@@ -0,0 +1,106 @@
+// Finally, peas. Base plant.
+/obj/item/seeds/peas
+	name = "pack of pea pods"
+	desc = "These seeds grows into vitamin rich peas!"
+	icon_state = "seed-peas"
+	species = "peas"
+	plantname = "Pea Vines"
+	product = /obj/item/reagent_containers/food/snacks/grown/peas
+	maturation = 3
+	potency = 25
+	growthstages = 3
+	growing_icon = 'icons/obj/hydroponics/growing_vegetables.dmi'
+	icon_grow = "peas-grow"
+	icon_dead = "peas-dead"
+	genes = list(/datum/plant_gene/trait/repeated_harvest)
+	mutatelist = list(/obj/item/seeds/peas/laugh) // TODO: Add /obj/item/seeds/peas/shoot at a later date, for the peashooter mutation line
+	reagents_add = list (/datum/reagent/consumable/nutriment/vitamin = 0.1, /datum/reagent/consumable/nutriment = 0.05, /datum/reagent/water = 0.05)
+	
+
+/obj/item/reagent_containers/food/snacks/grown/peas
+	seed = /obj/item/seeds/peas
+	name = "peapod"
+	desc = "Finally... peas."
+	icon_state = "peas"
+	filling_color = "#739122"
+	bitesize_mod = 1
+	foodtype = VEGETABLES
+	tastes = list ("peas" = 1, "chalky saltiness" = 1)
+	wine_power = 50
+	wine_flavor = "It tastes, distressingly, of fermented peas."
```

Comment:
might want to remove the "it tastes" part from this string since it'll look odd otherwise (It tastes It tastes, distressingly, of fermented peas)

Reply:
what kraso said